app/react-components/components/DownloadVisualizationButton.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import DropDownButton from './DropDownButton';
+import downloadSvg from '../utils/download-svg';
+import saveFile from '../utils/filesaver';
+import toTsvString, {mapArrayToTsvString} from '../utils/toTsvString';
+
+function getSelector(el) {
+  return typeof el === 'string' ? document.querySelector(el) : el;
+}
+const styles = {
+  hidden: {
+    width: 0,
+    height: 0,
+    overflow: 'hidden',
+  },
+};
+
+const icon = <i className={"fa fa-download"}/>;
```

Comment:
for icons, can you use the `react-icons` components as in https://github.com/NCI-GDC/portal-ui/blob/8978e029bb49839c425c9558807608e2c0f7c884/app/react-components/Project.js#L4